.github/workflows/ci.yml

```diff
@@ -10,6 +10,7 @@ on:
   push:
     branches:
       - "main"
+      - "staging-blast"
```

Comment:
Let's also add `staging`

Reply:
Added. If it looks good now, can you please approve?